adapt/engine.py

```diff
@@ -126,19 +126,23 @@ def determine_intent(self, utterance, num_results=1, include_tags=False, context
                     best_intent['__tags__'] = tags
                 yield best_intent
 
-    def register_entity(self, entity_value, entity_type, alias_of=None):
+    def register_entity(self, entity_values, entity_type, alias_of=None):
         """
         Register an entity to be tagged in potential parse results
 
         Args:
-            entity_value(str): the value/proper name of an entity instance (Ex: "The Big Bang Theory")
+            entity_values(str/arr): the value/proper nameS of an entity instances
+                    (Ex: "The Big Bang Theory", ["The Office", "Community"])
             entity_type(str): the type/tag of an entity instance (Ex: "Television Show")
         """
-        if alias_of:
-            self.trie.insert(entity_value.lower(), data=(alias_of, entity_type))
+        if type(entity_values) is not list: entity_values = [ entity_values ]
+	if alias_of:
```

Comment:
Is there an indentation problem here?

Reply:
Yes, there is. I seem to have broken it in between changing editors. More interestingly, it didn't break the code. Fixing it